src/vss_tools/vspec/vssexporters/vss2samm/helpers/namespaces.py

```diff
@@ -21,22 +20,20 @@ def get_node_name_from_vspec_uri(node_uri: URIRef):
 
 
 def get_unit_uri(unit_name: str):
-    return URIRef(f"{samm_base_namespace}:unit:{cfg.SAMM_VERSION}#{unit_name}")
+    return URIRef(f"{samm_base_namespace}:unit:{Config.SAMM_VERSION}#{unit_name}")
 
 
-log.debug("VSS to SAMM CONFIG:\n  -- SAMM_TYPE   : %s\n  -- SAMM_VERSION: %s\n", cfg.SAMM_TYPE, cfg.SAMM_VERSION)
-
 # NOTE: samm_base_namespace is more for the ESMF core libraries
 samm_prefix = "urn:samm"
 samm_base_namespace = f"{samm_prefix}:org.eclipse.esmf.samm"
 Namespaces = {
-    "samm": f"{samm_base_namespace}:meta-model:{cfg.SAMM_VERSION}#",
-    "samm-c": f"{samm_base_namespace}:characteristic:{cfg.SAMM_VERSION}#",
-    "samm-e": f"{samm_base_namespace}:entity:{cfg.SAMM_VERSION}#",
-    "unit": f"{samm_base_namespace}:unit:{cfg.SAMM_VERSION}#",
+    "samm": f"{samm_base_namespace}:meta-model:{Config.SAMM_VERSION}#",
+    "samm-c": f"{samm_base_namespace}:characteristic:{Config.SAMM_VERSION}#",
+    "samm-e": f"{samm_base_namespace}:entity:{Config.SAMM_VERSION}#",
+    "unit": f"{samm_base_namespace}:unit:{Config.SAMM_VERSION}#",
 }
 
 # Below formatted namespace should look like: urn:samm:com.covesa.vss.spec:5.0.0#
 # and is used for the ":" bindings of the converted to TTLs, VSS Aspect models
 # that will refer to the user specified output_namespace
-samm_output_namespace = f"{samm_prefix}:{cfg.OUTPUT_NAMESPACE}:{cfg.VSPEC_VERSION}#"
+samm_output_namespace = f"{samm_prefix}:{Config.OUTPUT_NAMESPACE}:{Config.VSPEC_VERSION}#"
```

Comment:
At this stage, all of OUTPUT_NAMESPACE, VSPEC_VERSION and SPLIT_DEPTH are not initialized i.e., they have Null values.